### DISCUSSION TITLE
Confirm Dialog template variables broken?

### DISCUSSION THREAD

**Discussion by epaitz** [primefaces/community#1230] [PrimeNG] [answered]
I was trying to use the Confirm Dialog template variables at v16 doing the following.

```
<p-button label="OK" (onClick)="onClick()"></p-button>
<p-confirmDialog>
    <ng-template pTemplate="message" let-confirmation>
        <p>{{confirmation == null}}</p>
    </ng-template>
</p-confirmDialog>

export class AppComponent {
    constructor(private confirmationService: ConfirmationService) {}
    onClick() {
        this.confirmationService.confirm({message: 'Hi'});
    }
}
```
When the dialog opens, I see "true" because the template variable, which I called confirmation, is null/undefined. I was assuming that I could do something like `<p>{{confirmation.message}}</p>`

There was an PR back in November 2023 that I am not sure was merged into the master branch correct. 
https://github.com/primefaces/primeng/pull/14154

Please let me know if this is an issue or if I am doing something wrong. 

Thanks

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example so we can identify the issue?

**Comment by epaitz** [comment]
Here is a stackblitz for Angular and PrimeNG v16. It seems to work ok in v17.
https://stackblitz.com/edit/github-y3azxc?file=package.json

**Reply by cetincakiroglu** [reply, marked as answer]
Hi,

We cannot provide a fix for v16 if it's not LTS, if it's possible please use the latest version.
